### PR TITLE
task-LkhxsAxM

### PR DESCRIPTION
**Pull request by BYavorskyj**
Added property: Links in menu blue but if you moved cursor over -link changed color to gray and removed underline
https://trello.com/c/LkhxsAxM/21-menu-change-hover
@AntonShalenui @vkryviak-ua

### CODE REVIEW

File: src/client/components/menu.css [hunk 1]
```diff
@@ -4,3 +4,10 @@ p {
 a {
     font-size: 14px;
 }
+.links :link {
```
Comment (vkryviak-ua) [inline src/client/components/menu.css]: What is `:link`

File: src/client/components/Menu.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ export default class Menu extends Component {
       <menu>
         <p>Menu</p>
         {
-          items.map(item => <p><a href={item.url}>{item.name}</a></p> )
+          items.map(item => <p className = "links"><a href={item.url}>{item.name}</a></p> )
```
Comment (vkryviak-ua) [inline src/client/components/Menu.js]: Plesa use link name.

File: src/client/components/menu.css (hunk 1 above)
Comment (vkryviak-ua) [reply src/client/components/menu.css]: And add menu selector:
`menu .link {`

File: src/client/components/menu.css [hunk 3]
```diff
@@ -4,3 +4,10 @@ p {
...
+    color: blue;
+}
+.links :hover {
```
Comment (vkryviak-ua) [inline src/client/components/menu.css]: Extra space `.link:hover {`

File: src/client/components/menu.css (hunk 1 above)
Comment (BYavorskyj) [reply src/client/components/menu.css]: changed

File: src/client/components/Menu.js (hunk 2 above)
Comment (BYavorskyj) [reply src/client/components/Menu.js]: did the other way

File: src/client/components/menu.css [hunk 4]
```diff
@@ -4,3 +4,10 @@ p {
 a {
     font-size: 14px;
 }
+.menu-text p a:link {
```
[the remark this answers is not in the document]
Comment (BYavorskyj) [reply src/client/components/menu.css]: ок

File: src/client/components/menu.css [hunk 5]
```diff
@@ -1,6 +1,12 @@
 p {
     font-size: 16px;
 }
+
 a {
     font-size: 14px;
 }
+
+.menu-text p a:hover {
```
Comment (vkryviak-ua) [inline src/client/components/menu.css]: Please add blue color
`.menu-text p a {
    color: blue;
}`

File: src/client/components/menu.css [hunk 6]
```diff
@@ -1,6 +1,16 @@
...
+    color: gray;
+    text-decoration: none;
+}
+
+.menu-text p a {
```
Comment (vkryviak-ua) [inline src/client/components/menu.css]: Usualy this style is before :hover

Comment (BYavorskyj) [reply src/client/components/menu.css]: ok

**Pull request merged**